src/api/favouriteFilters.test.tsx

```diff
@@ -65,4 +66,25 @@ describe('favourite filters api functions', () => {
       'sends axios request to edit favourite filters for a user and throws an appropriate error on failure'
     );
   });
+
+  describe('useDeleteFavouriteFilter', () => {
+    it('delete request to delete user session and returns successful response', async () => {
+      const { result } = renderHook(() => useDeleteFavouriteFilter(), {
+        wrapper: hooksWrapperWithProviders(),
+      });
+      expect(result.current.isIdle).toBe(true);
+
+      result.current.mutate('1');
+
+      await waitFor(() => {
+        expect(result.current.isSuccess).toBeTruthy();
+      });
+
+      expect(result.current.data).toEqual('');
+    });
+
+    it.todo(
+      'sends axios request to delete user favourite Filter and throws an appropriate error on failure'
+    );
```

Comment:
Since this is a mutation that actually has error handling attached to it already, the test doesn't need to be a TODO